aspnetcore/toc.md

```diff
@@ -131,21 +131,21 @@
 ## [Logging](xref:fundamentals/logging/index)
 ### [Logging with LoggerMessage](xref:fundamentals/logging/loggermessage)
 ## [Handle errors](fundamentals/error-handling.md)
-## [File Providers](fundamentals/file-providers.md)
 ## [Host](xref:fundamentals/host/index)
 ### [Web Host](xref:fundamentals/host/web-host)
 ### [Generic Host](xref:fundamentals/host/generic-host)
 ### [Background tasks with hosted services](xref:fundamentals/host/hosted-services)
-## [Session and app state](xref:fundamentals/app-state)
 ## [Servers](xref:fundamentals/servers/index)
 ### [Kestrel](xref:fundamentals/servers/kestrel)
 ### [ASP.NET Core Module](xref:fundamentals/servers/aspnet-core-module)
 ### [HTTP.sys](xref:fundamentals/servers/httpsys)
+## [Session and app state](xref:fundamentals/app-state)
+## [File Providers](fundamentals/file-providers.md)
```

Comment:
- Providers --> providers
- use xref link

Reply:
> Providers --> providers

The formal name of the technology is "File Providers." https://docs.microsoft.com/dotnet/api/microsoft.extensions.fileproviders